android/Samples/KMSample1/build.sh

```diff
@@ -75,6 +76,10 @@ fi
 if builder_start_action build:app; then
   ./gradlew clean $SAMPLE_FLAGS
 
-
   builder_finish_action success build:app
 fi
+
+if builder_start_action test:app; then
+
+  builder_finish_action success test:app
+fi
```

Comment:
We can technically skip this -- it won't error -- but it's not a bad idea to include it

```suggestion
if builder_start_action test:app; then
  # TODO: define tests
  builder_finish_action success test:app
fi
```